### DISCUSSION TITLE
OAuth2 after redirect: connect ECONNREFUSED 127.0.0.1:80

### DISCUSSION THREAD

**Discussion by kosch** [requarks/wiki#5594] [Help / Questions]
Environment:
- Container requarks/wiki:2.5.286 running as user. HAProxy in front of it, acting as TLS terminator.
- OAuth2 user directory configured for O365

When I try to login using OAuth2 and typing in the credentials for O365, the following redirect points to:
https://<wikijs host>/login/d13bd4bf-6683-4372-978b-c241f63020ec/callback?code=0.AXQA...
but ends up with an error by wikijs
Oops, something went wrong...
connect ECONNREFUSED 127.0.0.1:80

There is no interface 127.0.0.1 accessible for the container. 

Do I need to perform additonal configurations?

**Comment by matthew-mcdermott** [comment]
I have this exact same issue when trying to use Oauth2 with AWS Cognito. Any chance you found a solution to this issue? @kosch

**Comment by DomeDC** [comment]
I have this exact same issue when trying to use Oauth2 with Azure. 
If I choose the Azure method instead of the OAuthe2 method, I am no longer redirected back to Wiki.js 
it looks like the browser is stuck in a loop after a while i get the message from Microsoft: "A login is not possible, repeat the process" 
The method was implemented 1:1 according to the following instructions: https://docs.requarks.io/auth/azure
![image](https://user-images.githubusercontent.com/117277202/199478943-65db88f3-ef64-49df-8ab9-e79fb487cea9.png)

Thanks in advance for your help

greeting Dome (German)
